ParseUI-Widget/src/main/java/com/parse/ParseQueryAdapter.java

```diff
@@ -359,105 +368,37 @@ private void cancelAllQueries() {
    * {@code false}.
    */
   public void loadObjects() {
-    loadObjects(0, true);
+    loadNextPage(true);
   }
 
-  private void loadObjects(final int page, final boolean shouldClear) {
-    final ParseQuery<T> query = queryFactory.create();
-
-    if (objectsPerPage > 0 && paginationEnabled) {
-      setPageOnQuery(page, query);
+  private void loadNextPage(final boolean shouldClear) {
+    if (shouldClear && pager != null) {
+      cts.cancel();
```

Comment:
Since we set `cts` to `null` in `clear()`, do we need `null` checking for cts here?